frontend/src/leaderboard.js

```diff
@@ -1,6 +1,8 @@
 const ACCOUNTS_URL = 'http://localhost:3000/accounts';
 
 document.getElementById('leaderboard-button').addEventListener('click', () => {
+  document.getElementById('leaderboard-window').innerHTML = '';
```

Comment:
I thought we were supposed to try not to use `innerHTML`?

Reply:
I believe that was in reference to reading from things using innerHTML, since it is susceptible to users inputting malicious things. In this case I am simply clearing the element of any value, although I can check if there is a better/different way of doing it

Reply:
The other way that came to mind was removing it, but I still want to keep it on the page, just make it empty.

Reply:
Updated it to 
```suggestion
   const leaderboard = document.getElementById('leaderboard-window');
  while (leaderboard.firstChild) {
    leaderboard.removeChild(leaderboard.firstChild);
  }
```